pipeline_dp/mean_var_params.py

```diff
@@ -0,0 +1,132 @@
+"""DP computing of count, sum, mean, variance."""
+
+import numpy as np
+import pipeline_dp
+
+from dataclasses import dataclass
+
+
+@dataclass
+class MeanVarParams:
+    """Manages the parameters used for computing the mean/variance."""
```

Comment:
Nit: 
1.I'd suggest to drop "Manages" (w/o this work no change in meaning)
2.Please add computing the dp sum, count, mean, variance